test/testlib/minikube_utilities.go

```diff
@@ -451,6 +451,11 @@ func AwaitNrReplicasScheduled(t *testing.T, namespace string, expectedName strin
 		var podNames string
 		for _, pod := range FindAllPodsInSchema(t, namespace) {
 			if strings.Contains(pod.Name, expectedName) {
+				//ignore all completed pods
+				if pod.Status.Phase == corev1.PodSucceeded {
+					continue;
```

Comment:
Perhaps we should log the name of the completed pod, since this check is looking for continuously running pods in a workload and it is strange to find a completed job.

This is due to HAProxy introducing some kind of job that checks something having to do with their CRDs: https://github.com/haproxytech/helm-charts/commit/9baeddf7659ac9ea05cbc312db78ec3a199e40d7